cassandra/query.py

```diff
@@ -83,6 +84,29 @@ def tuple_factory(colnames, rows):
     """
     return rows
 
+class PseudoNamedTupleRow(object):
+    """
+    Helper class for pseudo_named_tuple_factory. Wraps a DefaultDict and
+    provides index- and attribute-based access to values, but otherwise does
+    not attempt to implement the full namedtuple interface.
+    """
+    def __init__(self, ordered_dict):
+        self._dict = ordered_dict
+        self._tuple = tuple(ordered_dict.values())
+
+    def __getattr__(self, name):
+        return self._dict[name]
+
+    def __getitem__(self, idx):
+        return self._tuple[idx]
+
+
+def pseudo_namedtuple_factory(colnames, rows):
+    """
+    """
```

Comment:
I'm, wondering if we really wants to make this public ... :thinking:

Reply:
What's the benefit to suggest users to use this instead of the dict_factory if they have the issue?

Reply:
This isn't given as a suggestion; [it's a fallback row factory](https://github.com/datastax/python-driver/pull/951/files#diff-6f3180b25cd5f084c6478485d9ca403eR156). I agree, there isn't any benefit to making this "public".

Reply:
Thoughts on this bit?